tests/test_nuheat.py

```diff
@@ -81,8 +81,14 @@ def test_get_request(self):
         self.assertEqual(response.status_code, 200)
         self.assertUrlsEqual(response.request.url, "{}?{}".format(url, urlencode(params)))
         request_headers = response.request.headers
-        self.assertEqual(request_headers["Origin"], config.REQUEST_HEADERS["Origin"])
-        self.assertEqual(request_headers["Content-Type"], config.REQUEST_HEADERS["Content-Type"])
+        self.assertEqual(
+            request_headers["Origin"],
+            util.get_request_headers()["Origin"],
```

Comment:
all header checks could be something like:

```suggestion
            api._request_headers["Origin"],
```